src/actions/PrescriptionActions.js

```diff
@@ -14,8 +14,59 @@ import { WizardActions } from './WizardActions';
 export const PRESCRIPTION_ACTIONS = {
   REFRESH: 'Prescription/refresh',
   FILTER: 'Prescription/filter',
+  OPEN_COMMENT_MODAL: 'Prescription/openCommentModal',
+  CLOSE_COMMENT_MODAL: 'Prescription/closeCommentModal',
 };
 
+const editTransactionCategory = newValue => (dispatch, getState) => {
+  const { prescription } = getState();
+  const { transaction } = prescription;
```

Comment:
Assuming this all required fields or safe getters? No defaults needed?

Reply:
ended up putting them in in a branched off pr hehe